opentech/static_src/src/app/src/containers/NoteListing.js

```diff
@@ -38,9 +41,14 @@ class NoteListing extends React.Component {
 
         if (submissionID && !isLoading) {
             loadNotes(submissionID);
+            this.pollNotes = setInterval(() => loadNotes(submissionID), 30000)
         }
     }
 
+    componentWillUnmount() {
+        this.pollNotes = null;
```

Comment:
We could use`clearInterval(this.pollNotes)` instead. That would be more proper I guess.